v4/broker/rabbitmq/rabbitmq.go

```diff
@@ -232,6 +233,10 @@ func (r *rbroker) Publish(topic string, msg *broker.Message, opts ...broker.Publ
 		m.Headers[k] = v
 	}
 
+	if r.getWithoutExchange() {
+		m.Headers["Micro-Topic"] = topic
```

Comment:
Could you replace the index with a reference to this constant; https://github.com/go-micro/go-micro/blob/68a6425ad85bfb74f9a9fce5c45ed3148fc2605b/transport/headers/headers.go#L6

Reply:
Oh wait never mind, this isn't contained in a release yet. Ignore this comment